test/unit/storage/storage.spec.ts

```diff
@@ -113,4 +113,22 @@ describe('Storage', () => {
       expect(storage.bucket('foo').name).to.equal('foo');
     });
   });
+
+  describe('Emulator mode', () => {
+    const EMULATOR_HOST = 'http://localhost:9199';
+
+    before(() => {
+      delete process.env.STORAGE_EMULATOR_HOST;
+      process.env.FIREBASE_STORAGE_EMULATOR_HOST = EMULATOR_HOST;
+    });
+
+    it('sets STORAGE_EMULATOR_HOST if FIREBASE_STORAGE_EMULATOR_HOST is set', () => {
+      expect(process.env.STORAGE_EMULATOR_HOST).to.equal(EMULATOR_HOST);
```

Comment:
I assume this relies on the `beforeEach()` block above calling `new Storage()`. I'd recommend making that call explicit here, so the test case is more self-contained.

```
new Storage(mockApp);
expect(process.env.STORAGE_EMULATOR_HOST).to.equal(EMULATOR_HOST);
```

Reply:
Done.